resources/home/dnanexus/utils/query.py

```
@@ -0,0 +1,487 @@
+            executor.submit(_find, report): report for report in reports
+        }
+
+        for future in concurrent.futures.as_completed(concurrent_jobs):
+            # access returned output as each is returned in any order
+            try:
+                data = future.result()
+
+                # Get info from the returned dict (will only ever be one
+                # clinical indication / CNV file dict)
+                sample = data["sample"]
+                clin_ind = list(data["clinical_indications"])[0]
```

Ruff (0.8.0)

Prefer `next(iter(data["clinical_indications"]))` over single element slice

Replace with `next(iter(data["clinical_indications"]))`

(RUF015)